.github/workflows/test-fakeredis.yml

```diff
@@ -0,0 +1,152 @@
+---
+name: Test Dragonfly/Fakeredis
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+
+
+concurrency:
+  group: dragon-fly-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        PYTHON_VERSION:
+          - "3.12"
+        DRAGONFLY_VERSION:
+          - "latest"
+        tests:
+          - "test_json"
+          #          - "test_mixins"
```

Comment:
Hi @cunla , thank you so much for this contribution :pray: 

Is there a reason that this is commented out ?

Reply:
Good catch! Forgot to uncomment it (this test is slower compared with the others and I wanted to test the github page creation).